department-of-veterans-affairs/va.gov-cms
# VACMS-7163: Delete the "VA Lovell Federal health care" menu

- Created ".install" with update hook to delete the "VA Lovell Federal health care" menu.

## Description

closes #7163 

## Testing done
Passes all tests

## Screenshots


## QA steps

As user administrator
1. [x] Go to `/admin/structure/menu?page=1` to verify and confirm that the "VA Lovell Federal health care" menu have been removed
2. [x] Go to the "Lovell Federal health care" page and enter node/edit mode `/node/15007/edit` to verify and check that the new "Lovell Federal health care" menu is still there


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
